rrfs-test/scripts/templates/run_fv3jedi_hera_template.sh

```diff
@@ -0,0 +1,36 @@
+#! /bin/sh
+#SBATCH --account=fv3-cam
+#SBATCH --qos=debug
+#SBATCH --ntasks=80
+#SBATCH -t 00:30:00
+#SBATCH --job-name=fv3jedi_test
+#SBATCH -o jedi.log
+#SBATCH --open-mode=truncate
+#SBATCH --cpus-per-task 2 --exclusive
+
+. /apps/lmod/lmod/init/sh
+set +x
+
+module purge
+
+module use @YOUR_PATH_TO_RDASAPP@/modulefiles
+module load RDAS/hera
```

Comment:
it should be "RDAS/hera.intel"  (for intel compiler).   The same  for mpasjedi one.

Reply:
@TingLei-NOAA, fixed.